agentstack/templates/proj_templates/graph.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "graph",
+    "description": "This is the start of your AgentStack project.",
+    "template_version": 4,
+    "framework": "langgraph",
+    "agents": [{
+        "name": "alex",
+        "role": "You are a friendly assistant.",
+        "goal": "Help the user with any of their requests.",
+        "backstory": "After years traveling the world, you've decided to get back into tech, just in time for the AI boom!. You're working on AgentStack, the fastest way to get started with AI agents. You have a README file available at: ./README.md",
+        "model": "openai/gpt-4o"
+    }],
+    "tasks": [{
+        "name": "hello_world",
+        "description": "As is tradition in software, let's start by saying 'Hello, World!'. Then, pick one or two tasks that they should try to do next with AgentStack.",
+        "expected_output": "The sentence Hello, World! followed by two things the user should try to customize their agent further.",
+        "agent": "alex"
+    }],
+    "tools": [{
+        "name": "file_read", 
+        "agents": ["alex"]
+    }],
+    "graph": [
```

Comment:
Putting down a thought for representing graph structures in template files. This isn't fully implemented yet.